src/get-own-property-symbols.js

```diff
@@ -1,12 +1,12 @@
-(function (Object, GOPS) {'use strict';
+function defineGetOwnPropertySymbols(Object, GOPS) {'use strict';
 
   // (C) Andrea Giammarchi - Mit Style
 
   if (GOPS in Object) return;
 
   var
     setDescriptor,
-    G = typeof global === typeof G ? window : global,
+    G = typeof global === "undefined" ? window : global,
```

Comment:
these are identical, what’s the point of this change?

Reply:
`G` is not defined before this usage. It would therefore always be `"undefined"`.

Reply:
right. so why not leave it as-is?

Reply:
It relies on the outcome of the call `typeof` of an undeclared variable. By stating decoratively this check is for checking of `undefined`ness, I would say this change makes the intent more clear.